wpiutil/src/main/native/include/wpi/sendable/SendableHelper.h

```diff
@@ -26,13 +26,22 @@ class SendableHelper {
   SendableHelper(const SendableHelper& rhs) = default;
   SendableHelper& operator=(const SendableHelper& rhs) = default;
 
+#if !defined(_MSC_VER) && !defined(__FRC_ROBORIO__)
+  // See https://bugzilla.mozilla.org/show_bug.cgi?id=1442819
+  __attribute__((no_sanitize("vptr")))
+#endif
   SendableHelper(SendableHelper&& rhs) {
     // it is safe to call Move() multiple times with the same rhs
     SendableRegistry::Move(static_cast<Derived*>(this),
                            static_cast<Derived*>(&rhs));
   }
 
-  SendableHelper& operator=(SendableHelper&& rhs) {
+#if !defined(_MSC_VER) && !defined(__FRC_ROBORIO__)
```

Comment:
This is because our Rio compiler doesn't have sanitize features enabled, or because it's too old?  If it's just because it's too old, this should be a version check instead.

Reply:
I tried it in godbolt: https://godbolt.org/z/3eGvvee1j. GCC 8 understands the attribute while GCC 7 does not.